draft-ietf-dhc-addr-notification.md

```diff
@@ -322,22 +322,24 @@ If an ADDR-REG-REPLY message is received for the address being registered, the c
 
 ## Registration Expiry and Refresh
 
-The client MUST refresh registrations to ensure that the server is always aware of which addresses are still valid. The client SHOULD perform refreshes as described below. Each refresh is scheduled for AddrRegRefresh seconds in the future, where AddrRegRefresh is min(4 hours, 80% of the address's current Valid Lifetime). Refreshes SHOULD be jittered by +/- 10% to avoid synchronization causing a large number of registration messages from different clients at the same time.
+The client MUST refresh registrations to ensure that the server is always aware of which addresses are still valid. The client SHOULD perform refreshes as described below.
 
-Whenever the client creates an address or receives a PIO which changes the Valid Lifetime of an existing address by more than 1%, then:
+We define a function AddrRegRefreshInterval(address) as min(4 hours, 80% of the address's current Valid Lifetime). When calculating this value, the client SHOULD apply a jitter of +/- 10% to avoid synchronization causing a large number of registration messages from different clients at the same time.
```

Comment:
Is the jitter fixed per client/per PIO? Otherwise we have a slight problem:

In theory,  as long as the network sends RAs with the same lifetime, the first RA creates an address, the second schedule a refresh, and all subsequent RAs do nothing until NextAddrRegRefreshTime comes (as min(now + AddrRegRefreshInterval, NextAddrRegRefreshTime) wouldn't change). However if you apply *dfferent* jitter every time when AddrRegRefreshInterval() is calculated, the client will be rescheduling the refresh there abd back again on *every* RA.

Reply:
Changed to be fixed per-client.